src/App.tsx

```diff
@@ -87,10 +90,15 @@ const App: FC = () => {
       <EngagementLink />
       <TreeNav />
       <Suspense fallback={<PageSpinner />}>
-        <Switch>
-          <Route path="/orgs/new" component={CreateOrgOverlay} />
-          <Route path="/orgs/:orgID" component={SetOrg} />
-        </Switch>
+        <div style={{height: '100%', width: '100%'}}>
```

Comment:
Might want to use a custom component like a Flexbox here, but all we really need is to make sure the globalheader is filling 100% of the available window, so that it's not artificially cut off - the UserProfile will be on the right side.